`TestGCOperations` is flaky

## Flaky Test

### Which jobs are failing
```
--- FAIL: TestGCOperations (2.85s)
    gc_test.go:353: 
        	Error Trace:	/home/runner/work/pd/pd/tests/server/gc/gc_test.go:353
        	Error:      	Not equal: 
        	            	expected: 3600
        	            	actual  : 3599
        	Test:       	TestGCOperations
```

### CI link
https://github.com/tikv/pd/actions/runs/17396617640/job/49379858276?pr=9671

### Reason for failure (if possible)


### Anything else

--- a/tests/server/gc/gc_test.go
+++ b/tests/server/gc/gc_test.go
@@ -350,7 +350,9 @@ func TestGCOperations(t *testing.T) {
 		re.Nil(resp.Header.Error)
 		re.Equal("b1", resp.GetDeletedBarrierInfo().GetBarrierId())
 		re.Equal(uint64(20), resp.GetDeletedBarrierInfo().GetBarrierTs())
-		re.Equal(3600, int(resp.GetDeletedBarrierInfo().GetTtlSeconds()))
+		// The TTL value decreases over time
+		re.Greater(resp.GetDeletedBarrierInfo().GetTtlSeconds(), int64(3595))
+		re.LessOrEqual(resp.GetDeletedBarrierInfo().GetTtlSeconds(), int64(3600))
 	}
 
 	for _, keyspaceID := range []uint32{constant.NullKeyspaceID, ks1.Id} {

